sdk/wallet-standard/src/features/iotaGetChainIdentifier.ts

```diff
@@ -0,0 +1,20 @@
+import { ChainIdentifier } from '../types';
+
+/**
+ * A Wallet Standard feature for retrieving the chain identifier from a wallet.
+ * This feature allows dApps to determine which network/chain the wallet is connected to.
+ */
+export type IotaGetChainIdentifierFeature = {
```

Comment:
Based on my comment about not using the genesis checkpoint digest, I suggest we rename to `IotaGetActiveChainFeature` and apply this convention everywhere